New version: voidtools.Everything version 1.4.1.1027

DandelionSprout opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/262725)

DandelionSprout committed:
New version: voidtools.Everything version 1.4.1.1027

DandelionSprout committed:
Improved the Scope values.

stephengillie commented:
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Installer https://www.voidtools.com/Everything-1.4.1.1027.ARM64.zip
 Scan request id: 'fb3151af-8d3b-4048-a022-580980347f13'
 
 Installer https://www.voidtools.com/Everything-1.4.1.1027.x64-Setup.exe
 Scan request id: '8acecd45-eb7d-4101-9817-fe2fca0c633f'

(Automated response - build 1061.)

DandelionSprout committed:
Update voidtools.Everything.installer.yaml

To cite Jeremy Clarkson: "How hard can it be?"

Thus far the outcome has been "Very."

DandelionSprout commented:
I figured, based on that the Everything package in #259336 seemingly being stuck indefinitely, I figured "It can't be that hard to get an Everything package to get validated, so as to break through the v(…).1027 barrier and ensure that any later updates get correctly handled by Winget as well."

Thus far the outcome has been "It is indeed that hard."

DandelionSprout commented:
I give up.
